mantle/kola/tests/rhcos/upgrade.go

```diff
@@ -357,15 +357,20 @@ func downloadLatestReleasedRHCOS(target string) (string, error) {
 
 	// The origin-clients package in Fedora doesn't `oc adm release info`
 	// ability.
+	binDir := fmt.Sprintf("%s/bin", target)
+	os.Mkdir(binDir, 0755)
 	ocUrl := fmt.Sprintf("https://mirror.openshift.com/pub/openshift-v4/%s/clients/ocp/latest/openshift-client-linux.tar.gz", system.RpmArch())
```

Comment:
Wait isn't this code completely unnecessary now that https://github.com/coreos/coreos-assembler/pull/2777 ?